doc/Caliptra.md

```diff
@@ -415,21 +415,171 @@ The owner key, when represented in fuses or in the FMC's alias certificate, is a
+| Validity                       | notBefore   | notBefore from firmware manifest
+|                                | notAfter    | notAfter from firmware manifest
+| Subject Name                   | CN          | Caliptra FMC Alias
+|                                | serialNumber | SHA256 hash of Subject public key
+| Subject Public Key Info        | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+|                                | Public Key  | FMC Alias Public Key value
+| Signature Algorithm Identifier | Algorithm   | ecdsa-with-SHA384
+|                                | Parameters  | Named Curve = prime384v1
+| Signature Value                |             | Digital signature for the certificate
+| KeyUsage                       | keyCertSign | 1
+| Basic Constraints              | CA          | True
+|                                | pathLen     | 3
+| Authority Key Identifier       |             | First 20 bytes of SHA256 hash of LDevID public key
+| tcg-dice-Ueid                  | ueid        | UEID specified by IDevID attribute fuses
+| tcg-dice-TcbInfo               | Flags       | NOT_CONFIGURED if lifecycle is unprovisioned
+|                                |             | NOT_SECURE if lifecycle is manufacturing
+|                                |             | DEBUG if not debug locked
+|                                | SVN         | concatenation of FMC SVN and FMC fuse SVN
+|                                | FWIDs       | [0] SHA384 digest of 
+|                                |             | lifecycle state
+|                                |             | debug locked state
+|                                |             | anti-rollback disable fuse
+|                                |             | ECDSA vendor public key index fuse
+|                                |             | LMS vendor public key index fuse
+|                                |             | LMS verification enable fuse
+|                                |             | boolean indicating whether owner public key hash is in fuses
+|                                |             | vendor public key hash
+|                                |             | owner public key hash
+|                                |             | [1] SHA384 digest of FMC
+
+Caliptra does not generate an Alias<sub>FMC</sub> CSR. Owners that wish to endorse Alias<sub>FMC</sub> must do so with proprietary flows.
+
+### Alias<sub>RT</sub> certificate
+Caliptra FMC generates the Alias<sub>RT</sub> certificate and endorses it with the Alias<sub>FMC</sub> private key. The Alias<sub>RT</sub> certificate implements the following field values:
+
+| Field                          | Sub Field   | Value
+| -------------                  | ---------   | ---------
+| Version                        | v3          | 2
+| Serial Number                  |             | First 20 bytes of SHA256 hash of Rt Alias public key
```

Comment:
Nit: "RT", all caps? Ditto just below.

Reply:
The cert builder has the subject name as "Caliptra Rt Alias" so left it at that. But I capitalized all the others. Should we change the code?

Reply:
Ah. Not super high priority I'd say.